docker-compose.yaml

```diff
@@ -0,0 +1,34 @@
+version: '3'
+services:
+  mongo:
+    image: mongo:3.4-xenial
+    ports:
+      - "27017"
+#  mongo-test:
+#    image: mongo:3.4-xenial
+#    links:
+#      - mongo
+#    depends_on:
+#      - mongo
+#    command: mongo test
+  elasticsearch:
+    image: elasticsearch:5.6
+    ports:
+      - "9200"
+#     - "9300"
+  uwazi:
+    build:
+      context: .
+      dockerfile: Dockerfile-devshell
+    volumes:
+      - .:/root/uwazi:Z,rw
+    ports:
+      - "3000"
+      - "8080"
+    depends_on:
+      - mongo
+      - elasticsearch
+    environment:
+      - DBHOST=mongo
+      - DATABASE_NAME=uwazi_development
```

Comment:
This line is critical in order to have our system point to the right database when DBHOST is configured.